docs/_includes/phone-input.html

```diff
@@ -0,0 +1,1105 @@
+                  <use href="#icon-tick-16"></use>
+                </svg>
+              </div>
+            </div>
+          </div>
+        </span>
+        <span class="textbox">
+          <span id="phone-prefix-5">+1</span>
+          <input class="textbox__control" type="tel" aria-label="area code + phone number"
+            autocomplete="tel-national" aria-describedby="phone-prefix-5"/>
+        </span>
+      </span>
+    </div>
+  </div>
+
+  {% highlight html %}
+  <span class="phone-input phone-input--error">
+    <span class="listbox-button listbox-button--form listbox-button--error">
+      <button class="btn btn--form btn--split-start" aria-expanded="false" aria-haspopup="listbox" type="button"
+        data-listbox-button-prefix="Country: ">
+        <span class="btn__cell">
+          <span class="btn__text">
+            <svg class="flag flag--us" focusable="false" height="18" width="24" aria-hidden="true" data-country-code="1"
+              aria-label="Country: United States of America (+1)">
+              <use href="#flag-us"></use>
+            </svg>
+          </span>
+          <svg class="icon icon--chevron-down-12" focusable="false" height="10" width="14" aria-hidden="true">
+            <use href="#icon-chevron-down-12"></use>
+          </svg>
+        </span>
+      </button>
+      <div class="listbox-button__listbox">
+        <div class="listbox-button__options" role="listbox">
+          <div class="listbox-button__option" role="option">
+            <span class="listbox-button__value">
+              <svg class="flag flag--al" focusable="false" height="18" width="24" aria-hidden="true" data-country-code="355">
+                <use href="#flag-al"></use>
+              </svg>
+              <span>Albania</span>
+              <span>(+355)</span>
+            </span>
+            <svg class="icon icon--tick-16" focusable="false" height="10" width="14" aria-hidden="true">
+              <use href="#icon-tick-16"></use>
+            </svg>
+          </div>
+          ...
+          <div class="listbox-button__option" role="option" aria-selected="true">
+            <span class="listbox-button__value">
+              <svg class="flag flag--us" focusable="false" height="18" width="24" aria-hidden="true" data-country-code="1">
+                <use href="#flag-us"></use>
+              </svg>
+              <span>United States of America</span>
+              <span>(+1)</span>
+            </span>
+            <svg class="icon icon--tick-16" focusable="false" height="10" width="14" aria-hidden="true">
+              <use href="#icon-tick-16"></use>
+            </svg>
+          </div>
+        </div>
+      </div>
+    </span>
+    <span class="textbox">
+      <span id="phone-prefix-5">+1</span>
+      <input class="textbox__control" type="tel" aria-label="area code + phone number"
+        autocomplete="tel-national" aria-describedby="phone-prefix-5"/>
```

Comment:
In this and in the example, we should add `aria-invalid=true`
Also how are we announcing to the user that listbox has an error? We should add a aria state to it show its erroring.

Reply:
Good catch, I updated the current error example with association.